mappings/cko.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS cko
+	METHOD a rescale ()V
```

Comment:
This method is called on MinecraftGame to update the GUI scale, no? Shouldn't it then be updateScale or something in this vein?

Reply:
yep, that's what its for. updateScale makes sense, will change to that